kart/lfs_util.py

```diff
@@ -21,10 +21,15 @@
 _EMPTY_SHA256 = "sha256:" + ("0" * 64)
 
 
+PRE_PUSH_HOOK = "\n".join(["#!/bin/sh", 'kart lfs+ pre-push "$@"', ""])
```

Comment:
```suggestion
PRE_PUSH_HOOK = "\n".join(["#!/bin/sh", 'exec kart lfs+ pre-push "$@"', ""])
```

the shell doesn't need to fork before running kart, it can just exec kart directly. Not sure if/how this runs on windows though

Reply:
Not done - I checked what the LFS hooks looked like on windows and macos and in both cases they looked like this... not really too concerned if the shell forks or not, but aiming to have a hook is at least as cross-platform as the original